deep_quoridor/src/plugins/wandb_train.py

```diff
@@ -78,23 +101,18 @@ def _initialize(self, game):
         }
         config.update(self.agent.model_hyperparameters())
 
-        self.run = wandb.init(
-            project=self.params.project,
-            job_type="train",
-            config=config,
-            tags=[self.agent.model_id(), f"-{self.params.run_id()}"],
-            id=self.params.run_id(),
-            group=f"{self.params.run_id()}",
-            notes=self.params.notes,
-        )
-        Timer.set_wandb_run(self.run)
+        self.run.config.update(config)
+        assert self.run.tags is not None
+        self.run.tags = self.run.tags + (self.agent.model_id(), f"-{self.params.run_id()}")
```

Comment:
Since we're now creating the wandb run before `start_game()` and `_initialize()` are called, here we just add the appropriate tags and config.